executor/index_merge_reader.go

```diff
@@ -359,8 +383,11 @@ func (w *partialTableWorker) extractTaskHandles(ctx context.Context, chk *chunk.
 			return handles, retChk, nil
 		}
 		for i := 0; i < chk.NumRows(); i++ {
-			h := kv.IntHandle(chk.GetRow(i).GetInt64(handleOffset))
-			handles = append(handles, h)
+			handle, err := handleCols.BuildHandleByOffsets(chk.GetRow(i), handleOffset)
```

Comment:
The handleCols has resolved column index, can we call `BuildHandle` without `handleOffset`?